package.json

```diff
@@ -11,7 +11,7 @@
     "yarn": "please-use-pnpm",
     "pnpm": ">=9"
   },
-  "packageManager": "pnpm@9.1.1",
+  "packageManager": "pnpm@9.1.0",
```

Comment:
Why are we downgrading? I recently upgraded my local setup to comply with this.

Reply:
cuz other core  repos are using pnpm 9.1.0, so we dont need to change pnpm version between repos

Reply:
framework and core-apps are using 9.1.0.